src/_posts/2019-05-10-initiative-circles.md

```diff
@@ -0,0 +1,37 @@
+---
+author: Jorge Gueorguiev Garcia
+layout: post
+asset-type: post
+title: "Initiative Circles at Codurance"
+date: 2019-05-10 08:00:00
+description: How Initiative Circles work at Codurance.
+abstract: Second post about internal workings of Codurance, concentrating on Initiative Circles.
+image: 
+    src: /assets/custom/img/blog/meeting2.png
+    attribution:
+       text: Photo by Sandro Viljoen
+tags: 
+- codurance
+---
+
+So this is the second post about the internal workings of Codurance. While in the previous one I looked at how salary and reviews work, here I want to concentrate on something that allows for me to take an active part in shaping our organisation: : Initiative Circles (IC from now on).
+
+I have always wanted to make my life and the life of my colleagues better. There is no company that will not benefit from improvements, some far more than others. At Codurance, with Initiative Circles, anyone has the opportunity to improve the lives of their co-workers, from small improvements to much more significant changes. At Codurance, through ICs, we have addressed disparate topics  such as: Paid Slack, BitWarden for password keeping, the Progress Structure that is the subject of my next post, salary alignment with the local market, , Peer Review Process, The Podcasts, …
+
+The Initiative Circles was an idea introduced by Sandro and Mash, our co-founders, with the vision to empower everyone to evolve the organisation according to the business and organisational needs. Their aim is to create a truly learning organisation. 
+
+IC are relatively simple to set up: someone has an itch to scratch, and floats the idea. If two other people like the idea and want to collaborate then the IC can proceed.  “Anything” can be tackled. You seek advice from people who will be impacted, reach a resolution, communicate to people, and go ahead and implement it. Rather than seeking complete consensus we work on trust that if enough people agree with an idea then the rest will trust them to have their best at heart. If people don’t like the results of an IC then they can create another IC and change its outcome.
```

Comment:
"change its outcome." -> "change it's outcome"

Reply:
Possesive, doesn't need apostrophe

Reply:
Doesn't it? I believe possessive nouns need apostrophes. In this case "it" is the IC, and it possesses the outcome. I may be missing something, but I did a quick google and I think it holds up my understanding: https://grammar.yourdictionary.com/punctuation/apostrophe-rules.html

Reply:
that doesn't talk about the pronouns her, his or ... its :-)
https://grammarist.com/spelling/its-its/

Reply:
Ahh, I forgot that "its" has an exception doesn't it. My bad. My dissertation supervisor used to catch me out on this one all the time 😅